assets/config/validator0.toml

```diff
@@ -1,62 +0,0 @@
-[loglevel]
```

Comment:
Files still referenced by `docker-compose.yml` and `scripts/main.sh`.

Reply:
docker-compose is all kinds of broken